apps/cloud-log/app.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+var DISABLE_LEVEL = require('@yoda/logger').UPLOAD_DISABLE_LEVEL
+var MIN_LEVEL = require('@yoda/logger').UPLOAD_MIN_LEVEL
+var MAX_LEVEL = require('@yoda/logger').UPLOAD_MAX_LEVEL
+var setGlobalUploadLevel = require('@yoda/logger').setGlobalUploadLevel
+var logger = require('@yoda/logger')('log-switch')
+var cloudgw = require('@yoda/cloudgw')
+
+module.exports = function (activity) {
+  activity.on('request', function (nlp, action) {
+    if (nlp.intent === 'RokidAppChannelForward') {
+      onCloudLogLevelSwitch(activity, nlp)
+    }
+  })
+}
+
+function onCloudLogLevelSwitch (activity, nlp) {
+  var level = parseInt(nlp.forwardContent.intent)
+  logger.info(`cloud logger switch to ${level}`)
+  if (level === DISABLE_LEVEL) {
+    setGlobalUploadLevel(level)
+  } else if (MIN_LEVEL <= level && level <= MAX_LEVEL) {
+    activity.get().then(config => {
```

Comment:
The config could be loaded once at the `create` event.

Reply:
emm... will the config be changed after switch user account?

Reply:
Correction: use `ready` event.